lib/API/JSON/Response.php

```diff
@@ -30,9 +30,10 @@ public function send() {
     if (!empty($this->meta)) {
       $response['meta'] = $this->meta;
     }
-    if ($data !== null) {
-      $response = array_merge($response, $data);
+    if ($data === null) {
+      $data = [];
     }
+    $response = array_merge($response, $data);
 
     if (!empty($response)) {
```

Comment:
This response is still an empty body because `[]` is considered empty here.

Reply:
How about allowing an empty array: `if (!empty($response) || is_array($response))`.

Reply:
https://circleci.com/gh/mailpoet/mailpoet/91305?utm_campaign=chatroom-integration&utm_medium=referral&utm_source=slack

I tried that but PHPStan is complaining that the condition is always true. And it is because `$response` is always an array. I think we could remove this condition altogether and send the response always. What do you think?

Reply:
Yes, that's an option.